### ISSUE TITLE
Configuration needs to be updated

### ISSUE THREAD

**Issue by matisku** [mrysz/homebridge-air-quality#1]
Newer version of Hombridge requires `name` parameter in configuration.
i.e.
```json
"accessories": [
    {
         "accessory":"AirQuality",
         "name":"Air Quality",
         "apikey":"YOUR_API_KEY",
         "latitude":"YOUR_LATITUDE",
         "longitude":"YOUR_LONGITUDE",
         "pollingInterval":300,
         "showAirQualityIndex":true,
         "airQualityIndexName":"Air Quality",
         "showHumidity":true,
         "humidityName":"Humidity",
         "showTemperature":true,
         "temperatureName":"Temperature"
    }
]
```

**Comment by mrysz** [comment]
Fixed in last commit 6729b5677992a72409f777421a2ad0ab7feac348

**Comment by matisku** [comment]
👍
